wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/BurstTimestampTableViewCell.swift

```diff
@@ -57,14 +58,15 @@ final class BurstTimestampSenderMessageCellDescription: ConversationMessageCellD
 
 final class BurstTimestampSenderMessageCell: UIView, ConversationMessageCell {
 
-    private let timestampView = ConversationCellBurstTimestampView()
+    private let timestampView: ConversationCellBurstTimestampView
     private var configuration: BurstTimestampSenderMessageCellConfiguration?
     private var timer: Timer?
 
     weak var delegate: ConversationMessageCellDelegate?
     weak var message: ZMConversationMessage?
 
     override init(frame: CGRect) {
+        self.timestampView = ConversationCellBurstTimestampView(userSession: ZMUserSession.shared()!)
```

Comment:
We shouldn't use `ZMUserSession.shared()`

Reply:
@johnxnguyen I know, this is temporary. If I tried to pass a userSession the app would crash.

Reply:
We need to make further changes here and I think it's not so straightforward.

Reply:
We fixed that as well.